Index: src/components/nav/Nav.jsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/components/nav/Nav.jsx b/src/components/nav/Nav.jsx
--- a/src/components/nav/Nav.jsx	
+++ b/src/components/nav/Nav.jsx	
@@ -32,35 +32,35 @@
                         <div className='text'>Home</div>
                     </a></li>
 
-                    <li onClick={() => setActiveNav({name: '#about', index: 1})} className={activeNav.name === '#about' ? 'active' : ''}><a href="#">
+                    <li onClick={() => setActiveNav({name: '#about', index: 1})} className={activeNav.name === '#about' ? 'active' : ''}><a href="#about">
                         <div className='icon'>
                             <img src={home} alt="home"/>
                         </div>
                         <div className='text'>About</div>
                     </a></li>
 
-                    <li onClick={() => setActiveNav({name: '#experience', index: 2})} className={activeNav.name === '#experience' ? 'active' : ''}><a href="#">
+                    <li onClick={() => setActiveNav({name: '#experience', index: 2})} className={activeNav.name === '#experience' ? 'active' : ''}><a href="#experience">
                         <div className='icon'>
                             <img src={home} alt="home"/>
                         </div>
                         <div className='text'>Experience</div>
                     </a></li>
 
-                    <li onClick={() => setActiveNav({name: '#portfolio', index: 3})} className={activeNav.name === '#portfolio' ? 'active' : ''}><a href="#">
+                    <li onClick={() => setActiveNav({name: '#portfolio', index: 3})} className={activeNav.name === '#portfolio' ? 'active' : ''}><a href="#portfolio">
                         <div className='icon'>
                             <img src={home} alt="home"/>
                         </div>
                         <div className='text'>Portfolio</div>
                     </a></li>
 
-                    <li onClick={() => setActiveNav({name: '#contact', index: 4})} className={activeNav.name === '#contact' ? 'active' : ''}><a href="#">
+                    <li onClick={() => setActiveNav({name: '#contact', index: 4})} className={activeNav.name === '#contact' ? 'active' : ''}><a href="#contact">
                         <div className='icon'>
                             <img src={home} alt="home"/>
                         </div>
                         <div className='text'>Contact</div>
                     </a></li>
 
-                    <li onClick={() => setActiveNav({name: '#language', index: 5})} className={activeNav.name === '#language' ? 'active' : ''}><a href="#">
+                    <li onClick={() => setActiveNav({name: '#language', index: 5})} className={activeNav.name === '#language' ? 'active' : ''}><a href="#language">
                         <div className='icon'>
                             <img src={home} alt="home"/>
                         </div>
